httpcore5-h2/src/main/java/org/apache/hc/core5/http2/impl/nio/AbstractH2StreamMultiplexer.java

```diff
@@ -1334,6 +1359,38 @@ H2Stream createStream(final H2StreamChannel channel, final H2StreamHandler strea
         return streams.createActive(channel, streamHandler);
     }
 
+    public final void sendPriorityUpdate(final int prioritizedStreamId, final PriorityValue value) throws IOException {
```

Comment:
@arturobernalg By the way this method does not seem to be used anywhere. Is this intended?

Reply:
@ok2c Yes, that was intentional. The logic was inlined into the stream submit path, so the standalone helper became redundant. I'm going to removed it.

Reply:
@arturobernalg Please let me know if you can live with #567. The functionality should be moved to client and we should put more work into the API design